guide/sections/global-services.adoc

```diff
@@ -117,3 +117,12 @@ TODO: to be completed
 Procedures pertinent to the Global Monitor will be described here once validated through testing during the WIS2 pilot phase.
 
 TODO: to be completed
+
+=== Global Registry ===
+
+* The roles of the Global Registry is to make available the information required for Global Infrastructure and WIS2 nodes to connect with each other. 
+* The Global Registry makes available to Global Caches the list of registered WIS2 Nodes. The following metadata is made available for each WIS2 Node: hostname and port(s) of local broker, center_id, list of implemented protocols (MQTT, MQTTs, web-socket).
```

Comment:
The Global Caches are not on the "front" of the connectivity. GBs are.
So, we could have "The Global Registry makes available to Global Services the list of registered WIS2 Nodes"